test/rendering/__fixtures__/scenario_2.fix.js

```diff
@@ -0,0 +1,1231 @@
+export default () => ({
+  type: 'sn-table',
+  genericObjects: [
+    {
+      getLayout: {
+        qInfo: {
+          qId: 'uJCJ',
+          qType: 'sn-table',
+        },
+        qMeta: {
+          privileges: ['read', 'update', 'delete', 'exportdata'],
+        },
+        qSelectionInfo: {},
+        qHyperCube: {
+          qSize: {
+            qcx: 1,
+            qcy: 6,
```

Comment:
you are right, there are more than 10 rows. But only qSize is checked which is below 10 @haxxmaxx

Reply:
why is qcy < 10 here? I think it is misleading that the code implies that the pagination should be hidden there are less than 10 rows, but the screenshots show otherwise

Reply:
just for the purpose of testing. The problem is that the  `qHyperCube` does not match with `getHyperCubeData` as they are from different places. Gonna work on it.

Reply:
Done